### DISCUSSION TITLE
Can't create repo error. Says too many files but project only has one file

### DISCUSSION THREAD

**Discussion by ghost** [git-lfs/git-lfs#5248] [Q&A]
I've tried creating a repo of a newly installed, empty unreal 5 project as well as a folder with a single text file. Both give same error: 

"Unable to create the new repository because there are too many new files in this directory"

Here's a video reproducing the issue: https://www.loom.com/share/65541b45c30346d182cd553869c17780

This was apparently an issue that was resolved back in 2019 but it looks like it's back? Here's the link to the previous thread talking about this: https://github.com/desktop/desktop/issues/7038

**Comment by bk2204** [comment]
Hey,

This sounds like a GitHub Desktop problem or a problem in some other software.  Git LFS doesn't create repositories and thus it doesn't produce this error message.  You'd probably be better off asking on the `desktop/desktop` repository's discussions or on Stack Overflow since it isn't related to this repository or project.
